src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/Cmdlets/UninstallPackageCommand.cs

```diff
@@ -95,7 +93,10 @@ protected override void ProcessRecordCore()
                 startTime,
                 _status,
                 _packageCount,
-                stopWatch.Elapsed.TotalSeconds);
+                stopWatch.Elapsed.TotalSeconds,
+                packageNamespaceEnabled: false,  // Packagenamespace doesn't matter when do uninstall.
```

Comment:
I don't think this is true in PackageReference. 

Uninstalling a top level package can lead to transitive package changes, such as version changes and by virtue of a version change, a package id change.

Reply:
Make sense, added correct value here.